doc.go

```diff
@@ -1,2 +1 @@
-// Package rdk TODO
```

Comment:
This is here because someone needs to write a valid doc comment for the package

Reply:
@edaniels Sorry for not providing more context-- had discussed with @mcvella and thought that this package description would be redundant with the rest of https://pkg.go.dev/go.viam.com/rdk, so decided to take out TODO from the page. Would you recommend I update this with a valid doc comment instead? can do so

Reply:
So this is currently visible on the go docs page: ![Screenshot 2023-05-18 at 19 33 15](https://github.com/viamrobotics/rdk/assets/5212232/1c6804af-69d9-4e93-8fab-2c810c09d682)